mediorum/server/placement.go

```diff
@@ -67,7 +83,9 @@ func (rh *RendezvousHasher) Rank(key string) []string {
 
 	// after migration complete, just call `rank`
 
+	rh.mu.Lock()
```

Comment:
is the theory here that this is getting interrupted somehow? that's crazy

Reply:
https://github.com/tysonmote/rendezvous/blob/master/rendezvous.go#L62

Reply:
MY GUY!